versioned_docs/version-8.7/self-managed/identity/deployment/configuration-variables.md

```diff
@@ -132,17 +132,92 @@ steps:
 Setting either of the feature flags to `true` requires a database connection. To configure a database
 connection, see [database configuration](#database-configuration).
 :::
 
+## Multi-tenancy
+
+Multi-tenancy in the context of Camunda 8 refers to the ability of Camunda 8 to serve multiple distinct tenants or clients within a single installation. For more information, including additional configuration requirements, see the [multi-tenancy documentation](/docs/self-managed/operational-guides/configure-multi-tenancy.md).
+
+The following configuration is required to enable multi-tenancy in Identity:
+
+| Environment variable                    | Description                                                                                                               | Default value |
+| --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `CAMUNDA_IDENTITY_MULTITENANCY_ENABLED` | Controls the multi-tenancy feature within Identity. This can be set in helm via `camunda.tasklist.multi-tenancy.enabled`. | `false`       |
+
+With multi-tenancy enabled, the following variables can be used to configure additional [tenants](/self-managed/identity/user-guide/tenants/managing-tenants.md). To add additional tenants, increment the `0` value for each variable (for example, `IDENTITY_TENANTS_1_NAME`).
+
+| Environment variable                         | Description                                                                                                                                                                                                                                        | Default value |
+| -------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `IDENTITY_TENANTS_0_NAME`                    | A human-readable name for the tenant.                                                                                                                                                                                                              | Default       |
+| `IDENTITY_TENANTS_0_TENANTID`                | The tenant ID. The `<default>` tenant is automatically created during Identity startup.                                                                                                                                                            | `<default>`   |
+| `IDENTITY_TENANTS_0_MEMBERS_0_TYPE`          | The type of member to add to the tenant, one of `GROUP`, `USER`, or `APPLICATION`. Additional members can be added by incrementing the second `0` value (for example, `IDENTITY_TENANTS_0_MEMBERS_1_TYPE`.)                                        | `APPLICATION` |
```

Comment:
I had to backward engineer this a bit. Are these the only available types?

Reply:
That's correct, the valid values are here in code https://github.com/camunda-cloud/identity/blob/4b0b00105170daebdf65352d707b2b43995119d2/management-api/src/main/java/io/camunda/identity/entity/AccessRule.java#L102-L106